scripts/lintmigrations_custom.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+import sys  # To get the absolute path to the currently running Python interpreter
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from pathlib import Path
+
+BASE_DIR = Path(__file__).parent.parent
+
+# Extract the app label from the directory structure
+def get_app_labels():
+    return [
+        path.parent.name
+        for path in Path(BASE_DIR / "cms").glob("*/migrations")
+    ]
+
+
+# Run the lintmigrations command for a given app_label
+def run_lintmigration(app_label):
```

Comment:
Suggestion: It looks like `lintmigrations` supports a `--include-apps` flag, which can be passed multiple times. That might be better to run than running the command in parallel.

Reply:
Regarding adding the linter only during development, will make that change. As just needed to make sure of some things/ it was a temporary thing we did to make things work =)

Reply:
I think that comment might be related to my other one. In this case, you're using `concurrent.futures` to run the command in parallel. If you instead pass `--include-apps`, you can avoid the concurrency and likely end up with a briefer output anyway.

The auto-discovery of modules would remain.

Reply:
Yes

Reply:
Yh, please ignore the comments from Friday afternoon 😄 

@RealOrangeOne, it seems that if you run the linter with either `--include-apps` or  `--exclude-apps`  it still prints the log lines for the apps it ignores, so our suggestions wouldn't give us what we want in terms of readability.

However - I think we can ignore all of that, because there's an option that allows you to skip the log lines based on their type. See my suggested change below and the option from the library's documentation:
<!DOCTYPE html>
--quiet or -q {ok,ignore,warning,error} | Suppress certain output messages, instead of writing them to stdout.
-- | --

Reply:
Interesting. Using `-q ignore` ought to avoid the need for a custom script entirely, as it makes the output far more readable. If not, then using `--include-apps` should still avoid the threading complexity.

Reply:
`poetry run python manage.py lintmigrations --quiet ignore ` - we can use this instead (this still uses `exclude_apps` in `dev.py`) and yes, would mean we don't need the script . The output being:
`(analysis, 0001_initial)... WARNING
        CREATE INDEX prolongs transaction, delaying lock release
(bundles, 0001_initial)... WARNING
        CREATE INDEX prolongs transaction, delaying lock release
(core, 0001_initial)... WARNING
......
(themes, 0001_initial)... WARNING
        CREATE INDEX prolongs transaction, delaying lock release
(topics, 0001_initial)... WARNING
        CREATE INDEX prolongs transaction, delaying lock release
(users, 0001_initial)... WARNING
        CREATE INDEX prolongs transaction, delaying lock release
*** Summary ***
Valid migrations: 1/206
Erroneous migrations: 3/206
Migrations with warnings: 13/206
Ignored migrations: 189/206
` 
As more migrations are made, this may not be an ideal output but shall I change the command to this?

Reply:
That sounds like it's going to be much easier to maintain. The list might get long, but it shows each migration as being valid, which is something. it might also be an idea to add "ok" to the quiet flag, so it only prints out the errors (making it even quieter).